Gemfile.lock

```diff
@@ -47,10 +57,11 @@ GEM
     in_threads (1.6.0)
     method_source (1.0.0)
     mini_portile2 (2.8.4)
-    minitest (5.18.1)
+    minitest (5.20.0)
```

Comment:
> NameError: uninitialized constant MiniTest

Gotta update all `MiniTest::Spec` to `Minitest::Spec` re: this bump if I recall correctly.

Reply:
Just opened a PR to perform the `minitest` upgrade with `MiniTest` ➡ `Minitest` migration.
- https://github.com/freeCodeCamp/devdocs/pull/2103

Reply:
:wave: @simon04, should be able to request `renovate` to rebase this PR to fix it up re: the following. :rocket:
- https://github.com/freeCodeCamp/devdocs/pull/2103

Reply:
@larouxn, thanks for the remainder. Rebase done manually.